project/Dependencies.scala

```diff
@@ -53,6 +56,7 @@ object Dependencies {
   val sprayJson     = "io.spray"        %% "spray-json"    % "1.3.1"
 
   val apacheMath    = "org.apache.commons" % "commons-math3" % "3.2"
+  val scalaBreeze   = "org.scalanlp"  %%  "breeze"  % "0.11.2"
```

Comment:
I think Breeze has more Linear Algebra type algorithms in it; that's my impression anyway. But we should definitely see if spire fills any requirements and go with that if we can.